src/Data/THGen/XML.hs

```diff
@@ -311,11 +313,12 @@ isoXmlGenerateEnum (ExhaustivenessName strName' exh) enumCons = do
 isoXmlGenerateRecord :: PrefixName -> [IsoXmlDescRecordPart] -> TH.DecsQ
 isoXmlGenerateRecord (PrefixName strName' strPrefix') descRecordParts = do
   let
+    isNewtype     = length descRecordParts == 1
```

Comment:
Something like `hasSingleField` would be more clear

Reply:
This check was used at least twice: once to clarify how to construct a type definition, an another to supply fields in a different manner. The second usage differs for newtype/data, but have no meaningful difference for 1/more fields. This is why I find the name more appropriate.